pkgs/tools/graphics/astc-encoder/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "astc-encoder";
+  version = "2.5";
+
+  src = fetchFromGitHub {
+    owner = "ARM-software";
+    repo = "astc-encoder";
+    rev = version;
+    sha256 = "0ff5jh40w942dz7hmgvznmpa9yhr1j4i9qqj5wy6icm2jb9j4pak";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  # Built with CMake flags from https://github.com/ARM-software/astc-encoder/blob/main/Docs/Building.md
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DISA_NONE=ON"
+  ] ++ lib.optionals stdenv.hostPlatform.isx86_64 [
+    "-DISA_AVX2=ON"
+    "-DISA_SSE41=ON"
+    "-DISA_SSE2=ON"
+  ] ++ lib.optionals stdenv.hostPlatform.isAarch64 [
+    "-DISA_NEON=ON"
+  ];
+
+  # Link binaries into environment and provide 'astcenc' link
+  postInstall = ''
+    mv $out/astcenc $out/bin;
+    ln -s $out/bin/astcenc-none $out/bin/astcenc;
+  '' + lib.optionalString stdenv.hostPlatform.isx86_64 ''
+    ln -sf $out/bin/astcenc-sse2 $out/bin/astcenc;
+  '';
+
+  outputs = [ "out" ];
```

Comment:
```suggestion
```

Reply:
ah ok, good to know that this is the default.